external/include/headers/spirv.h

```diff
@@ -50,12 +50,12 @@
 
 typedef unsigned int SpvId;
 
-#define SPV_VERSION 100
-#define SPV_REVISION 1
+#define SPV_VERSION 10000
```

Comment:
Gah.  Should file an issue upstream to indicate this likely should be a hex value ?
I believe the "1" part of it  should be mapping to the upper 16 bits of the generator word.